sql/updates/reverse-dev.sql

```diff
@@ -75,3 +75,28 @@ BEGIN
     );
 END
 $$ LANGUAGE 'plpgsql';
+
+--undo compression feature for caggs
+--check that all continuous aggregates have compression disabled
+--this check is sufficient as we cannot have compressed chunks, if
+-- compression is disabled
+DO $$
+DECLARE
+  cagg_name NAME;
+BEGIN
+    FOR cagg_name IN
+        SELECT view_name, 
+               materialization_hypertable_schema ,
+               materialization_hypertable_name   
+        FROM timescaledb_information.continuous_aggregates
+        WHERE compression_enabled is TRUE
+    LOOP
+        RAISE NOTICE 'Compression is enabled for continuous aggregate: %', cagg_name; 
+    END LOOP;
+    RAISE EXCEPTION 'Cannot downgrade as compression is enabled for continuous aggregates' 
```

Comment:
I think this new logic is broken because you always issue an exception after the loop... actually you should raise the exception only it some cagg with compression is found on the previous loop, no?

Reply:
You are too quick with the reviews :-) Just caught that with my local testing and modified.